vault-core/src/main/java/org/apache/jackrabbit/vault/fs/io/AbstractExporter.java

```diff
@@ -398,6 +400,28 @@ private static PackageType detectPackageType(WorkspaceFilter filter)  {
         return PackageType.MIXED;
     }
 
+    /**
+     * Returns the version of an Oak module.
+     *
+     * @param moduleName the name of the module
+     * @param clazz a class of the module
+     * @return the version (or "SNAPSHOT" when unknown)
+     */
+    private String getVersion(String moduleName, Class<?> clazz) {
+        // borrowed from oak-commons
+        String version = "SNAPSHOT"; // fallback
+        try (InputStream stream = clazz
+                .getResourceAsStream("/META-INF/maven/org.apache.jackrabbit.vault/" + moduleName + "/pom.properties")) {
+            if (stream != null) {
+                Properties properties = new Properties();
+                properties.load(stream);
+                return properties.getProperty("version", version);
+            }
+        } catch (IOException ignored) {
```

Comment:
When is this supposed to happen? Why should we ignore that?

Reply:
Well, we are reading from a stream.

I think best effort is perfectly ok here (but we could log the exception).